### PR TITLE
more content and translation to forum.php

### PR DESCRIPTION
**Pull request by HWSKY**
this file was originally translated according to /lang/en/forum.php root file
all translations and dependencies are fully included in this file since the last change, commit: 625f425 
sorry for the mess on pull requests, i will try send the single commits in single pulls

### CODE REVIEW

**Comment by HWSKY** [comment]
I passed for a long revision to this modification, all seems ok now.
2 grammatical errors solved.

File: resources/lang/pt-br/forum.php [hunk 1]
```diff
@@ -89,21 +124,53 @@
...
         ],
 
         'moderate_move' => [
             'title' => 'Mover para outro fórum',
         ],
 
+        'pin' => [
+            'pin-0' => 'Desprender tópico',
+            'pin-1' => 'Prender tópico',
+            'pinned-0' => 'Tópico foi desprendido',
```
Comment (nanaya) [inline resources/lang/pt-br/forum.php]: this has been renamed to `state-0`

File: resources/lang/pt-br/forum.php [hunk 2]
```diff
@@ -32,6 +32,17 @@
             'confirm' => 'Você tem certeza que quer remover esta imagem de capa?',
         ],
     ],
+
+
+
+
+
```
Comment (nanaya) [inline resources/lang/pt-br/forum.php]: why are there so many newlines

File: resources/lang/pt-br/forum.php [hunk 3]
```diff
@@ -89,21 +124,53 @@
...
+            'lock-1' => 'Tópico trancado',
+            'state-0' => 'Tópico foi destrancado',
+            'state-1' => 'Tópico foi trancado',
         ],
 
         'moderate_move' => [
             'title' => 'Mover para outro fórum',
         ],
 
+        'pin' => [
```
Comment (nanaya) [inline resources/lang/pt-br/forum.php]: also renamed to `moderate_pin`

File: resources/lang/pt-br/forum.php (hunk 1 above)
Comment (UnusualToast) [inline resources/lang/pt-br/forum.php]: These should be "Desfixar tópico"/"Fixar tópico"/"Tópico foi desfixado".

File: resources/lang/pt-br/forum.php [hunk 4]
```diff
@@ -47,8 +58,8 @@
         'post_number_input' => 'digite número do post',
         'total_posts' => ':posts_count posts no total',
     ],
-    'subforums' => 'Subfórums',
-    'title' => 'osu!community',
+    'subforums' => 'Sub-fórums',
```
Comment (UnusualToast) [inline resources/lang/pt-br/forum.php]: Still wrong, haha. Should be "Sub-fóruns" or "Subfóruns". Not an actual word in Portuguese, but it still should respect general grammar rules when borrowed from English.

**Pull request closed without merging**